packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js

```diff
@@ -117,8 +118,13 @@ export default class BackupService {
   }
 
   getCatalog(ovhSubsidiary) {
-    return this.catalogApi.get({ catalogName: 'privateCloud', ovhSubsidiary })
-      .$promise;
+    return this.$http
+      .get(`2api/sws/dedicatedcloud/catalog`, {
```

Comment:
```js
get('/sms/dedicatedcloud/catalog', {
  serviceType: 'aapi',
  params: { ... },
})

Reply:
@frenautvh I can confirm that the URL is the correct one.

Reply:
You mean passing the `serviceType` parameter?

Reply:
yes i though we had to set the serviceType but in core, an interceptor is setting it automatically if your url starts with 2api so i guess it's fine :)
see : https://github.com/ovh/manager/blob/master/packages/manager/modules/core/src/index.js#L176

Reply:
Thanks for pointing out this.

I've added the `serviceType` parameter and remove extra template strings.